python/python/bystro/api/auth.py

```diff
@@ -12,7 +12,6 @@
 
 if not os.path.exists(DEFAULT_DIR):
     os.makedirs(DEFAULT_DIR, exist_ok=True)
-    print(f"Created default credentials path: {CREDENTIALS_PATH}")
```

Comment:
when running from Docker, this message would pop up on every ancestry score cli run, and is completely unhelpful.